wicket-examples/src/main/java/org/apache/wicket/examples/websocket/JSR356Application.java

```diff
@@ -59,11 +65,22 @@ public void init()
 
 		getSharedResources().add(ChartWebSocketResource.NAME, new ChartWebSocketResource());
 
+		final WebSocketSettings webSocketSettings = new WebSocketSettings() {
+			@Override
+			public void configureSession(IWebSocketSession webSocketSession) {
+				LOGGER.info("getMaxIdleTimeout = " + webSocketSession.getMaxIdleTimeout());
+				// make sessions almost "immortal"
+				webSocketSession.setMaxIdleTimeout(Duration.ofHours(10).toMillis());
```

Comment:
Is this a good idea ?
We hosts the examples at https://examples9x.wicket.apache.org/websockets/

Reply:
@martin-g I saw locally disconnection happening after a few millisecond of inactivity. Mind connection will be closed once user navigates away from page. One interesting fact is that specification states that 0 or negative means for ever, but jetty does not accepts negative values (it throws an exception) and 0 seems to be really 0. I will double check. What value would you use here?

Reply:
5 mins at most. This is just a demo, we don't guarantee 99.999% SLA :-)